### DISCUSSION TITLE
In Azure, the tab does not update automatically once the report is ready

### DISCUSSION THREAD

**Discussion by documentat-alibaev** [allure-framework/allure2#2366] [Bug Reports] [answered]
1. I enable the Allure Report plugin for Azure DevOps.
2. I add an Allure Report step to the pipeline.
3. I start a pipeline.
4. I go to the “Allure Report” tab of the running job.
5. The tab displays a loading spinner, since the report is not ready yet.
6. The job finishes successfully.
7. The spinner still shows, and the report never appears unless I refresh the page.

**Comment by epszaw** [comment, marked as answer]
Hey there! In new version (1.0.7) of the extension, the problem should be solved. Please, try the latest version
